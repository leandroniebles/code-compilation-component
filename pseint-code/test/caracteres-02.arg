--allow_accents=0