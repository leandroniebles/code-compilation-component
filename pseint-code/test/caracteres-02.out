=== Line 2: SynError 68
=== Line 2: SynError 65
=== Line 3: SynError 68
=== Line 3: SynError 65
=== Line 4: SynError 68
=== Line 4: SynError 65
=== Line 5: SynError 68
=== Line 5: SynError 65
=== Line 6: SynError 68
=== Line 6: SynError 65
=== Line 7: SynError 68
=== Line 7: SynError 65
=== Line 8: SynError 68
=== Line 8: SynError 65
=== Line 9: SynError 68
=== Line 9: SynError 65
=== Line 10: SynError 68
=== Line 10: SynError 65
=== Line 11: SynError 68
=== Line 11: SynError 65
=== Line 12: SynError 68
=== Line 12: SynError 65
=== Line 13: SynError 68
=== Line 13: SynError 65
=== Line 14: SynError 68
=== Line 14: SynError 65
=== Line 15: SynError 68
=== Line 15: SynError 65
=== Line 16: SynError 68
=== Line 16: SynError 65
=== Line 17: SynError 68
=== Line 17: SynError 65
